tools/eksDistroBuildToolingOpsTools/docker/Dockerfile

```diff
@@ -9,4 +9,6 @@ ADD $BINARY_OUTPUT_PATH /usr/bin
 
 RUN chmod +x /usr/bin/$BINARY_NAME
```

Comment:
Why does this need the chmod? isnt this a built go binary?

Reply:
That is a god question. It is the binary so I don't think we need it which could mean we use the git minimal image, right?